libs/components/forms/testing/src/checkbox/checkbox-harness.spec.ts

```diff
@@ -9,7 +12,11 @@ async function setupTest(
   options: { dataSkyId?: string; hideEmailLabel?: boolean } = {},
 ) {
   await TestBed.configureTestingModule({
-    imports: [CheckboxHarnessTestModule],
+    imports: [
+      CheckboxHarnessTestModule,
+      NoopAnimationsModule,
```

Comment:
If popovers needs `NoopAnimationsModule`, I would put it in the `exports` of `SkyHelpTestingModule` so that we don't have to import `NoopAnimationsModule` every time we want to test help popovers.